modin/experimental/core/execution/native/implementations/hdk_on_native/calcite_serializer.py

```diff
@@ -133,8 +133,10 @@ def serialize_item(self, item):
             return self.serialize_obj(item)
         elif isinstance(item, list):
             return [self.serialize_item(v) for v in item]
+        elif isinstance(item, dict):
+            return {k: self.serialize_item(v) for k, v in item.items()}
 
-        self.expect_one_of(item, str, int)
+        self.expect_one_of(item, str, int, type(None))
```

Comment:
I suppose the function can return None now? In this case, need to update the docstring.